UI/auth-youtube.cpp

```diff
@@ -368,6 +378,9 @@ YoutubeChatDock::YoutubeChatDock(const QString &title) : BrowserDock(title)
 			 &YoutubeChatDock::SendChatMessage);
 	QWidget::connect(sendButton, &QPushButton::pressed, this,
 			 &YoutubeChatDock::SendChatMessage);
+
+	// Ensure local state for `isLoggedIn` is updated after UI loads.
+	QTimer::singleShot(1000, this, &YoutubeChatDock::YoutubeCookieCheck);
```

Comment:
Should be superfluous here as `YoutubeCookieCheck` is executed automatically before the URL of the browser dock is changed, or did this not happen before the first URL change in your tests?

Reply:
Okay, removed this.

Reply:
Did it work as expected? I hadn't tested if this was still needed, hence why I formulated it as a question.

Reply:
I was previously trying to account for the case of an existing login and subsequent restart of the OBS app, initializing the API based "send message" support for YouTube Live Chat (as opposed to using the native input panel, provided by the signed-in chat window). This was also when I previously had a bug in the cookie check logic that was reloading the page. With that bug resolved, this now works as expected, even without the initialization bit. Thanks for checking :-)